lib/releases.sh

```diff
@@ -8,7 +11,14 @@ function get_latest_release() {
     release="$(curl -H "Authorization: token ${GITHUB_TOKEN}" -sL "${1}")" || ( set -x && exit 1 )
   fi
   # This gets the latest release as vx.y.z , ignoring any version with a suffix starting with - , for example -rc0
-  release_tag="$(echo "$release" | jq -r "[.[].tag_name | select( startswith(\"${2:-""}\")) | select(contains(\"-\")==false)] | max ")"
+  if [[ $OS == ubuntu ]]; then
+    release_tag="$(echo "$release" | jq -r "[.[].tag_name | select( startswith(\"${2:-""}\")) | select(contains(\"-\")==false)]" \
+      | cut -sf2 -d\"  | tr '.' ' ' | sort -n +1 +2 | tail -n 1 | tr ' ' '.')"
+  else
+    release_tag="$(echo "$release" | jq -r "[.[].tag_name | select( startswith(\"${2:-""}\")) | select(contains(\"-\")==false)]" \
+      | cut -sf2 -d\"  | tr '.' ' ' | sort -nk1 -nk2 -nk3 | tail -n 1 | tr ' ' '.')"
+  fi
```

Comment:
would the CentOS one work on Ubuntu ? If it is different version of sort there might be some compatibility in one way. You could also do something like this to reduce code repetition: 
```suggestion
  release_tags_unsorted="$(echo "$release" | jq -r "[.[].tag_name | select( startswith(\"${2:-""}\")) | select(contains(\"-\")==false)]" \
      | cut -sf2 -d\"  | tr '.' ' ' )"
  if [[ $OS == ubuntu ]]; then
    release_tags_sorted="$(echo "$release_tags_unsorted"  | sort -n +1 +2 )"
  else
    release_tags_sorted="$(echo "$release_tags_unsorted" | sort -nk1 -nk2 -nk3 )"
  fi
  release_tag="$(echo "$release_tags_sorted" | tail -n 1 | tr ' ' '.')"
```

Reply:
I liked the proposed way of sorting, it is cleaner compared to what I have. Updated it. Thanks @maelk !